src/main/java/edu/tamu/iiif/service/AbstractManifestService.java

```diff
@@ -165,10 +165,22 @@ protected Model getRdfModel(String url) throws NotFoundException {
     }
 
     protected String getRdf(String url) throws NotFoundException {
-        Optional<String> rdf = Optional.ofNullable(restTemplate.getForObject(url, String.class));
-        if (rdf.isPresent()) {
-            logger.debug("RDF for {}: \n{}\n", url, rdf.get());
-            return rdf.get();
+        try {
+            if (logger.isDebugEnabled()) {
+                logger.info("Requesting RDF for {}", url);
```

Comment:
Is `logger.debug()` somehow not working?
I would think `logger.debug()` only runs when debug is enabled, helping us avoid the need to explicitly check `logger.isDebugEnabled()`.

Reply:
In this case I only want an info log when debug is enabled.